clients/java/src/main/java/io/zeebe/client/impl/ZeebeClientImpl.java

```diff
@@ -315,13 +296,28 @@ public CompleteJobCommandStep1 newCompleteCommand(final long jobKey) {
     return jobClient.newCompleteCommand(jobKey);
   }
 
+  @Override
+  public CompleteJobCommandStep1 newCompleteCommand(final ActivatedJob job) {
+    return jobClient.newCompleteCommand(job);
```

Comment:
I had expect that we just delegate here to jobkey. Like job.getKey(). Then the other methods are not necessary.